public/components/register_model/__tests__/register_model_artifact.test.tsx

```diff
@@ -97,6 +100,6 @@ describe('<RegisterModel /> Artifact', () => {
     await result.user.click(result.submitButton);
 
     expect(urlInput).toBeInvalid();
-    expect(onSubmitMock).not.toHaveBeenCalled();
+    expect(onSubmitWithURLMock).not.toHaveBeenCalled();
```

Comment:
Do we need to add test case for submitWithURL successfully?

Reply:
You mean for the function `submitWithURL` itself?

Reply:
Yes. Fro the test case part, we won't expect the `onSubmitWithURLMock` has been called before.

Reply:
Added one more unit test to cover form submit with `onSubmitWithURLMock`